### ISSUE TITLE
Training the model without optical flows

### ISSUE THREAD

**Issue by ioankont** [edurnebernal/SST-Sal#1]
Hello and thank you for your great job. Could you please tell me about the changes to the model, the optimizer and the learning rate you used for training without optical flows? thanks

**Comment by edurnebernal** [comment]
We used the following hyperparameters: 

- stochastic gradient descent optimizer with a learning rate of 0.001
-  batch size = 1
- epochs = 240
- frame input resolution = 320x240
- sequence length = 20 frames

The model's architecture has undergone slight adjustments due to a change in the input shape, which is now a tensor with dimensions of 320x240x3 instead of 320x240x6. As a result, we have reduced the network's hidden dimension to 9. To replicate or training, you can instantiate the model like this:  `model = models.SST_Sal(input_dim=3, hidden_dim=9)`.

**Comment by ioankont** [comment]
Thank you so much!
